services/workflows-service/src/env.ts

```diff
@@ -76,6 +76,26 @@ export const serverEnvSchema = {
     .optional()
     .describe('Bucket name of Data migration folders'),
   NOTION_API_KEY: z.string().describe('Notion API key'),
+  SECRETS_MANAGER_PROVIDER: z
+    .enum(['aws-secret-manager'])
+    .default('aws-secret-manager')
+    .describe('Secrets Manager provider'),
+
+  // AWS Secrets Manager
+  AWS_SECRETS_MANAGER_PREFIX: z
+    .string()
+    .optional()
+    .default('/dev/customers')
```

Comment:
how do you replace prefix according the env?
I suggest Prefix to have only `/customers`

Reply:
You must have the env name as well because our primary AWS account is shared between sb and prod.